src/v0/destinations/braze/deleteUsers.js

```diff
@@ -43,27 +53,16 @@ const userDeletionHandler = async (userAttributes, config) => {
       };
 
       const resp = await httpSend(requestOptions);
-
-      if (resp && !resp.success && !resp.response.response) {
-        throw new CustomError(
-          resp.response.code || "Could not delete user",
-          400
-        );
-      }
-      if (!resp || !resp.response) {
-        throw new CustomError("Could not get response", 500);
-      }
-      if (
-        resp &&
-        resp.response &&
-        resp.response.response &&
-        resp.response.response.status !== 200 &&
-        resp.response.response.status !== 404
```

Comment:
Previously, `404` is treated as a success.

Reply:
This is fixed